pcsx2/gui/Panels/GameDatabasePanel.cpp

```diff
@@ -430,15 +434,21 @@ bool Panels::GameDatabasePanel::WriteFieldsToDB() {
 	if (serialBox->GetValue().IsEmpty()) return false;
 
 	Game_Data game;
+	wxString com = compatBox->GetValue();
 	GameDB->findGame(game, serialBox->GetValue());
 
 	game.id = serialBox->GetValue();
 
-	game.writeString(L"Serial",		serialBox->GetValue());
-	game.writeString(L"Name",		nameBox->GetValue());
-	game.writeString(L"Region",		regionBox->GetValue());
-	game.writeString(L"Compat",		compatBox->GetValue());
-	game.writeString(L"[comments]",	commentBox->GetValue());
+	game.writeString(L"Serial", serialBox->GetValue());
+	game.writeString(L"Name", nameBox->GetValue());
+	game.writeString(L"Region", regionBox->GetValue());
+	if (com == "Unknown")          game.writeString(L"Compat", "0");
```

Comment:
@willkuer there is no standard string type, how do you expect it to even work (with wxString in this case)?

This code is readable and understandable so there is no reason to reinvent a wheel trying to "beautify" and optimize it.